### ISSUE TITLE
Going to Tidy Up Code

### ISSUE THREAD

**Issue by joshocurry** [joshocurry/DeepLearningAssignment1#1]
Next step is to breakdown big Gradient Descent Function into many different functions in order to tidy up code, and perform logistic regression on both Datasets on blackboard.

**Comment by joshocurry** [comment]
Cleaned Up, Still need to incorporate new DataSets from Blackboard

**Comment by joshocurry** [comment]
- Datasets loaded and code ran
- accuracy on linearly separable data is 1.0
- accuracy on non-linearly sep data at 0.5
- Need to apply train_test_split to ensure not overfitting

**Comment by joshocurry** [comment]
- Once NN applied, acc of non-lin data gone up => seems to work, quite inefficient.
- Still need to apply train.....
